S/stb/build_tarballs.jl

```diff
@@ -0,0 +1,33 @@
+# Build script for stb: C++ header-only single-file utilities.
+# This is basically just a dummy package that installs header files, and makes
+# them available for other packages.
+
+using BinaryBuilder
+
+
+name = "stb"
+# stb doesn't ever do releases, so the best we can do is stick to a specific git
+# revision...
+version = v"0.0.20221025"
+sources = [
+    ArchiveSource("https://github.com/nothings/stb/archive/8b5f1f37b5b75829fc72d38e7b5d4bcbf8a26d55.zip",
+                  "93a16ee3e866e719feec459f6f132cce932c5ec751eb38e3ec1975f911353d2e")
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/stb*/
+mkdir -p "${includedir}"
+cp -vr include/* "${includedir}/."
```

Comment:
I think it's
```suggestion
cp -vr *.h "${includedir}/."
```
?

Reply:
Yes, it looks like the tarball https://www.mlpack.org/files/stb.tar.gz has a different tree structure, so here we need to do things differently.  Here there are way more files.

Reply:
Thanks, you beat me to it again! :)